src/canopy_cli/cli.py

```diff
@@ -132,6 +136,22 @@ def _load_kb_config(config_file: Optional[str]) -> Dict[str, Any]:
     return kb_config
 
 
+def _validate_chat_engine(config_file: Optional[str]):
+    config = _read_config_file(config_file)
+    Tokenizer.initialize()
```

Comment:
We should have a "with tokenizer..."
A nice feature

Reply:
Yeah, thought about it.  
Doesn't worth the effort right now...

Reply:
Just task, i will add